src/main/webapp/WEB-INF/bb-manifest.xml

```diff
@@ -48,7 +48,7 @@
             <permission type="java.lang.RuntimePermission" name="db.connection.bb_bb60" actions="persist" />
             <permission type="java.lang.reflect.ReflectPermission" name="suppressAccessChecks" />
             <permission type="java.util.PropertyPermission" name="*" actions="read" />
-            <permission type="java.io.FilePermission" name="$java_home/-" actions="read"/>
+            <permission type="java.io.FilePermission" name="${java.home}/-" actions="read"/>
             <permission type="java.io.FilePermission" name="BB_HOME/-" actions="read,write,delete"/>
             <permission type="java.io.FilePermission" name="BB_HOME/logs/plugins/-" actions="write" />
```

Comment:
Betekent dat `-` erachter niet 'alle andere paden eronder' en is die vorige regel dus eigenlijk niet al genoeg? Enig idee?

Reply:
Ja datzelfde vroeg ik mij ook af, maar ik liet het er "voor de zekerheid" maar in staan, maar inderdaad, de docs van `java.io.FilePermission` zegt het volgende over het `-` teken:
```
A pathname that ends with "/-" indicates (recursively) all files and subdirectories contained in that directory. 
```
Ik denk eerlijk gezegd dat het ook zonder die line zal werken dus. Ik haal hem eruit en we zien het denk ik wel bij de test. Dan maar hopen dat als het niet werkt, dat de exception dan duidelijk genoeg is dat het om het `BB_HOME/logs/plugins/` pad gaat.